spec/support/query_counter.rb

```diff
@@ -5,10 +5,11 @@ def self.count(&block)
     new.count(&block)
   end
 
-  IGNORED_STATEMENTS = %w(CACHE SCHEMA)
+  IGNORED_STATEMENTS = %w(CACHE SCHEMA).freeze
+  IGNORED_QUERIES    = /^(?:ROLLBACK|BEGIN|COMMIT|SAVEPOINT|RELEASE|SAVEPOINT)/.freeze
```

Comment:
SAVEPOINT is in here twice.

Also, I don't think freezing the regex, nor the Array, saves you anything, does it?  (more curious if it does anything at all since it's not a key in a Hash or anything. @matthewd @tenderlove @jrafanie?)